Add list of integrations to CSE Automation Service Beta doc (#2026)
SumoLogic/sumologic-documentation: docs/cse/integrations/automation-service.md

* Playbooks type must be **CSE**.
* The CSE Automation Service only supports automated Enrichment, Notification, and Custom Action types at this time. 
* Actions can run "on premise" via a Bridge or can run directly through the Cloud. For security and performance reasons, only Certified Integrations/Actions can run directly through the Cloud; custom Actions must run "on premise".

> * Cloud SOAR App Central, where you can browse the full Integration and Playbook catalog, is not yet connected to the CSE Automation Service. A selection of popular Integrations have been added to your environment automatically, but the full list of available Integrations is included at bottom of this document. Contact your Sumo Logic account representative if you would like to have one of these Integrations added to your environment, or if you're interested in an Integration that's not listed.

Reviewer: ```suggestion
* Cloud SOAR App Central, where you can browse the full Integration and Playbook catalog, is not yet connected to the CSE Automation Service. A selection of popular Integrations have been added to your environment automatically, but the full list of available Integrations is included in [Available Integrations](#available-integrations) below. Contact your Sumo Logic account representative if you would like to have one of these Integrations added to your environment, or if you're interested in an Integration that's not listed.
```
